types/pica/pica-tests.ts

```diff
@@ -0,0 +1,34 @@
+let picaOptions: PicaOptions = { features: ['js', 'wasm', 'ww', 'cib'] };
+let resizer: Pica = new Pica(picaOptions);
```

Comment:
Just suggestions, as the options are optional, could we also include the parameterless constructor test as well? Exactly as documented on the package readme (all ctros are parameterless there):
```js
const pica = Pica();
pica.resize(img, canvas).then(...);
```
Thanks!

Reply:
Added tests with parameterless constructor